packages/editor/src/components/block-compare/block-view.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * WordPress dependencies
+ */
+import { Button } from '@wordpress/components';
+
+const BlockView = ( { title, rawContent, renderedContent, action, actionText, className } ) => {
+	return (
+		<div className={ className }>
+			<div className="editor-block-compare__content">
+				<h1 className="components-modal__header-heading">{ title }</h1>
+
+				<div className="editor-block-compare__html">
+					{ rawContent }
+				</div>
+
+				<div className="editor-block-compare__preview edit-post-visual-editor">
+					{ renderedContent }
+				</div>
+			</div>
+
+			<div className="editor-block-compare__action">
+				<Button isLarge tabindex="0" onClick={ action }>{ actionText }</Button>
```

Comment:
I'm guessing a console warning is being logged on this line?

We should pass `tabIndex` as the DOM property `tabIndex={ 0 }`, not the attribute name.

>Note that you should still use the canonical React naming for known attributes:
>
>```
>// Yes, please
><div tabIndex="-1" />
>
>// Warning: Invalid DOM property `tabindex`. Did you mean `tabIndex`?
><div tabindex="-1" />
>```
>
>In other words, the way you use DOM components in React hasn’t changed, but now you have some new capabilities.

https://reactjs.org/blog/2017/09/08/dom-attributes-in-react-16.html

Reply:
Oddly it doesn't log a warning, but both very valid points and I'll get those fixed in another PR